### ISSUE TITLE
13.4 with --turbo expects appDir experimental

### ISSUE THREAD

**Issue by dirtycajunrice** [vercel/next.js#49258]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Pro
    Binaries:
      Node: 18.12.1
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 13.4.0
      eslint-config-next: 13.3.0
      react: 18.2.0
      react-dom: 18.2.0
```

### Which area(s) of Next.js are affected? (leave empty if unsure)

App directory (appDir: true), Turbopack (--turbo)

### Link to the code that reproduces this issue

base template can reproduce

### To Reproduce

base template can reproduce

### Describe the Bug

When running dev --turbo in 13.4 without experimental appDir definition it fails with
```
throw new Error("> The `app` directory is experimental. To enable, add `appDir: true` to your `next.config.js` configuration under `experimental)
```

### Expected Behavior

To not check an unnecessary flag

**Comment by tony1658** [comment]
If it's intentional I would suggest changing the error message because it would be misleading:
> The `app` directory in combination with TurboPack is experimental. To enable, add `appDir: true` to your `next.config.js` configuration under `experimental

Otherwise yeah remove this error.

**Comment by dirtycajunrice** [comment]
I believe this is the issue line
https://github.com/vercel/next.js/blob/43c8c1bb9a471aa0a6340cb44d47d5f1b0d4216e/packages/next/src/lib/turbopack-warning.ts#LL21C14-L21C25

**Comment by tony1658** [comment]
And this https://github.com/vercel/next.js/blob/fd319d3922178ec31ef5a7a70fc9a46397f160fb/packages/next/src/lib/find-pages-dir.ts#L41#L50

**Comment by alexkirsz** [comment]
Fixed in https://github.com/vercel/next.js/pull/49241.

This fix will go out as part of v13.4.1. In the meantime, it's available under [v13.4.1-canary.2](https://github.com/vercel/next.js/commit/2c37ec01c58e019800e54d03d3e9b76393989968).
